src/train_lm.py

```diff
@@ -349,6 +353,7 @@ def compute_objective(metrics: Dict[str, float]) -> float:
     
     if training_args.calculate_grad_variance:
         grad_callback = GradientVarianceCallback(eval_dataset_tokenized,
+                                                 keys=training_args.grad_keys.split(','),
```

Comment:
Perhaps you should do the split inside the callback, and just pass a string here? So that someone reading the code doesn't have to keep track of how the args are transformed when they should be just passed

Reply:
I think it makes sense to leave it as it is or make dict of keys instead of list